README.rst

```diff
@@ -137,6 +137,41 @@ or, clone the repository directly from GitHub:
   cd exoctk
   python setup.py [install|devlop]
 
+Obtain the ``exoctk`` Data
```

Comment:
I notice that this title is slightly larger than the one for `Obtain the ``exoctk`` Package`.  Is that intentional?

Reply:
Nice catch. That is not intentional, let me take a look.

Reply:
@bourque Turns out my terminal text is too smoll to see the difference between `~` and `-`. Should look better now.